tests/cli/test_run.py

```diff
@@ -132,6 +133,7 @@ def test_valid_zuliprc_but_no_connection(
 
     lines = captured.out.strip().split("\n")
     expected_lines = [
+        "Detected Platform: " + PLATFORM,
```

Comment:
Note that this will be the platform that the tests are running on. We may want to mock this instead to ensure we get some known output (and avoid the running vs testing confusion for readers).

It's good to keep tests in the same commit as the code that changes, as it's a unit of change, so here combined with one of the other commits.

We may want a specific test to handle how this output should vary on different platforms, including unsupported.